calcite-adapter/src/main/java/software/amazon/documentdb/jdbc/calcite/adapter/DocumentDbMethod.java

```diff
@@ -25,8 +25,10 @@
  * Builtin methods in the MongoDB adapter.
  */
 public enum DocumentDbMethod {
-    MONGO_QUERYABLE_FIND(DocumentDbTable.DocumentDbQueryable.class, "find", String.class,
+    // TODO: Investigate using find() for simpler queries.
```

Comment:
Let's have GitHub issue open for any TODO items, and reference them from code.

Reply:
New feature request added! https://github.com/aws/amazon-documentdb-jdbc-driver/issues/240